### PR TITLE
Handling tree retrieval timeout

### PR DESCRIPTION
**Pull request by cloderic**

### CODE REVIEW

File: craftai/client.py [hunk 1]
```diff
@@ -55,6 +61,9 @@ def config(self, cfg):
                                     """ or invalid owner provided.""")
     if not isinstance(cfg.get("operationsChunksSize"), six.integer_types):
       cfg["operationsChunksSize"] = 200
+    if (cfg.get("decisionTreeRetrievalTimeout") is not False and
```
Comment (Giska) [inline craftai/client.py]: if cfg.get("decisionTreeRetrievalTimeout") and .... ?

File: craftai/client.py [hunk 2]
```diff
@@ -311,28 +317,61 @@ def get_decision_tree(self, agent_id, timestamp):
 
     return decision_tree
 
+  def get_decision_tree(self, agent_id, timestamp):
+    # Raises an error when agent_id is invalid
+    self._check_agent_id(agent_id)
+
+    if self._config["decisionTreeRetrievalTimeout"] is False:
```
Comment (Giska) [inline craftai/client.py]: if not self._config["decisionTreeRetrievalTimeout"]

File: scripts/update_version.sh [hunk 3]
```diff
@@ -102,7 +102,7 @@ echo "Increment version from v$current_major.$current_minor.$current_patch to v$
 TODAY=`date +%Y-%m-%d`
 
 eval "sed -i.bak 's/$current_major.$current_minor.$current_patch/$next_major.$next_minor.$next_patch/g' $INIT_PY_FILE"
-eval "sed -i.bak 's/.*[[Unreleased]].*/## [Unreleased](https:\/\/github.com\/$GH_ORG\/$GH_REPO\/compare\/v$next_major.$next_minor.$next_patch...HEAD) ##$NL$NL## [$next_major.$next_minor.$next_patch](https:\/\/github.com\/$GH_ORG\/$GH_REPO\/compare\/v$current_major.$current_minor.$current_patch...v$next_major.$next_minor.$next_patch) - $TODAY ##/g' $CHANGELOG_MD_FILE"
+eval "sed -i.bak 's/.*\[Unreleased\].*/## [Unreleased](https:\/\/github.com\/$GH_ORG\/$GH_REPO\/compare\/v$next_major.$next_minor.$next_patch...HEAD) ##$NL$NL## [$next_major.$next_minor.$next_patch](https:\/\/github.com\/$GH_ORG\/$GH_REPO\/compare\/v$current_major.$current_minor.$current_patch...v$next_major.$next_minor.$next_patch) - $TODAY ##/g' $CHANGELOG_MD_FILE"
```
Comment (Giska) [inline scripts/update_version.sh]: :+1:

File: craftai/client.py (hunk 1 above)
Comment (cloderic) [reply craftai/client.py]: by design

File: craftai/client.py (hunk 2 above)
Comment (cloderic) [reply craftai/client.py]: by design

**Pull request merged**